tests/UI/pages/BO/catalog/discounts/index.js

```diff
@@ -292,6 +304,107 @@ class CartRules extends BOBasePage {
       wantedStatus ? this.bulkEnableLink : this.bulkDisableLink,
     );
   }
+
+  /* Pagination methods */
+  /**
+   * Get pagination label
+   * @param page
+   * @return {Promise<string>}
+   */
+  getPaginationLabel(page) {
+    return this.getTextContent(page, 'ul.pagination.pull-right li.active a');
```

Comment:
Can't add a selector ?

Reply:
Yes, please use a selector because if one day it breaks we'll have to look manually in the code. If it's in a selector, everything is in the same place and it's easier to check it.

Reply:
Done thx :+1: